_config.yml

```diff
@@ -42,11 +42,11 @@ c4sponsorsUrl: "mailto:conference@swiftfest.io"
 
 # Navigation
 navigationLinks:
- - {permalink: "/", text: "Home"}
+# - {permalink: "/", text: "Home"}
 # - {permalink: "/schedule/", text: "Schedule"}
  - {permalink: "/speakers/", text: "Speakers"}
  - {permalink: "/team/", text: "Team"}
- - {permalink: "/logistics/", text: "Logistics"}
+ - {permalink: "#footer", text: "FAQ"}
```

Comment:
I'm not so sure about this change. It cleans up the menu, which is great, but since the footer is way at the bottom, when we scroll to it, it's not exactly obvious where we should be looking. But I guess we can try it, and if we start getting a lot of questions, we can change it back.